client/post-editor/media-modal/detail/detail-item.jsx

```diff
@@ -324,7 +326,7 @@ const connectComponent = connect( state => {
 		isJetpack: isJetpackSite( state, siteId ),
 		isVideoPressEnabled: getSiteOption( state, siteId, 'videopress_enabled' ),
 		isVideoPressModuleActive: isJetpackModuleActive( state, siteId, 'videopress' ),
-		isPrivateSite: isPrivateSite( state, siteId ),
+		isPrivate: isPrivateSite( state, siteId ),
```

Comment:
Was this renamed to avoid `no-shadow` lint errors? I like that `isPrivateSite` is unambiguous, we know from the naming that it is the site that is private. Perhaps `isSitePrivate` would maintain the expressiveness yet avoid variable shadowing.

Reply:
It was, yes, and I also didn't like dropping site from it, but for some reason my brain was blind to swapping the words around. This is much better!